dojo/tools/wpscan/parser.py

```diff
@@ -71,13 +71,50 @@ def get_findings(self, file, test):
                 else:
                     dupes[dupe_key] = finding
 
+        # Wordpress version findings
+        if tree.get('version', [])['vulnerabilities']:
+            for vul in tree.get('version', [])['vulnerabilities']:
+                description = "\n".join([
+                    '**Title:** `' + vul['title'] + "`\n",
+                ])
+                finding = Finding(
+                    title=vul['title'],
+                    description=description,
+                    severity='High',
```

Comment:
This is artificial data. other parsers set it to `Medium` by default, why do you want to set it to `High`?

Reply:
Since plugin related findings were marked 'Medium', wanted to mark version related findings as 'High'. But to keep it consistent with others parsers, setting severity to 'Medium'